### PR TITLE
test(docparser): add unitary test for parseJSONTag

### PR DESCRIPTION
**Pull request by kratisto**

### CODE REVIEW

**Comment by alexjomin** [comment]
I'll give a look asap

File: docparser/parser_test.go [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+			expectedJSONTag: jsonTagInfo{
+				required: true,
+				name:     "jsontagname",
+				enum:     []string{"a", "b"},
+			},
+		},
+	}
+	for _, tc := range testCases {
+		t.Run(tc.description, func(t *testing.T) {
+			parsedJSONTag, err := parseJSONTag(tc.field)
```
Comment (alexjomin) [inline docparser/parser_test.go:141]: I should prefer go the "standard lib" way :

```go
if (err != nil) != (len(tc.expectedError) > 0) {
    t.Fatalf("got error = %v, wantErr %v", err, tc.expectedError)
}

if !reflect.DeepEqual(tc.expectedJSONTag, parsedJSONTag) {
    t.Errorf("got %v, want %v", parsedJSONTag, tc.expectedJSONTag)
}
```

**Review by alexjomin — CHANGES_REQUESTED** [review]
Thank you for this PR \o/

For this project I would like to use only the standard lib for testing. So please find below my comment.

Cheers.

**Comment by kratisto** [comment]
@alexjomin I updated the PR

**Pull request merged**